src/components/PodcastCard/PodcastCard.js

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+PodcastCard.propTypes = {
+  image: PropTypes.string.isRequired,
+  title: PropTypes.string.isRequired,
+}
+
+export default function PodcastCard({ image, title }) {
+  return (
+    <Card>
+      <PodcastImage src={image} alt="Podcast" />
```

Comment:
First I wanted to write something like "get rid of the `alt="Podcast"` and go with an empty alt-attribute". But on the other hand telling the user that this is a podcast might not be so bad as well ... 🤔